Create script which allows for a smarter install

Right now we are utilizing this by specifying a clone on the `postinstall` of our app `"scripts": {
    "postinstall": "git clone https://github.com/kethinov/wait-for-it.git node_modules/wait-for-it" }`, the problem is if you run `npm i` and it's already installed (cloned), the `npm i` fails because the directory already exists.

We should abstract this clone into another file, which can check directories and what not before doing the clone so the install step doesn't fail.

Output from `npm i`

```
C:\Users\alallier\Documents\something>npm i

> chromedriver@2.33.0 install C:\Users\alallier\Documents\something\node_modules\chromedriver
> node install.js

Downloading https://chromedriver.storage.googleapis.com/2.33/chromedriver_win32.zip
Saving to C:\Users\alallier\AppData\Local\Temp\chromedriver\chromedriver_win32.zip
Received 781K...
Received 1566K...
Received 2350K...
Received 3134K...
Received 3918K...
Received 4125K total.
Extracting zip contents
Copying to target path C:\Users\alallier\Documents\something\node_modules\chromedriver\lib\chromedriver
Done. ChromeDriver binary available at C:\Users\alallier\Documents\something\node_modules\chromedriver\lib\chromedriver\chromedriver.exe

> roosevelt@0.9.4 postinstall C:\Users\alallier\Documents\something\node_modules\roosevelt
> node ./lib/configAuditor.js

📋  Starting roosevelt user configuration audit...
✔  Configuration audit completed with no errors found.

> something@0.1.0 postinstall C:\Users\alallier\Documents\something
> git clone https://github.com/kethinov/wait-for-it.git node_modules/wait-for-it

fatal: destination path 'node_modules/wait-for-it' already exists and is not an empty directory.
npm ERR! code ELIFECYCLE
npm ERR! errno 128
npm ERR! something@0.1.0 postinstall: `git clone https://github.com/kethinov/wait-for-it.git node_modules/wait-for-it`
npm ERR! Exit status 128
npm ERR!
npm ERR! Failed at the something@0.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\alallier\AppData\Roaming\npm-cache\_logs\2017-12-12T17_31_36_166Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -1,0 +1,10 @@
+{
+  "name": "wait-for-it",
+  "author": "Giles Hall",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "repository": {
+    "private-repo": "git+https://github.com/kethinov/wait-for-it"
+  }
+}

